cmd/opera/launcher/launcher.go

```diff
@@ -116,6 +116,7 @@ func initFlags() {
 		validatorPubkeyFlag,
 		validatorPasswordFlag,
 		SyncModeFlag,
+		utils.GCModeFlag,
```

Comment:
Let's make a copy of `utils.GCModeFlag` but with default value = `archive`. It's because some existing nodes rely on the absence of GC, and will be overwhelmed if we implicitly enable GC

Reply:
okay, I got it